gitea/release.go

```diff
@@ -46,6 +47,24 @@ func (c *Client) GetRelease(user, repo string, id int64) (*Release, error) {
 	return r, err
 }
 
+// ListReleaseAssets gets all the assets of a release in a repository
+func (c *Client) ListReleaseAssets(user, repo string, id int64) (*Release, error) {
+	r := new(Release)
+	err := c.getParsedResponse("GET",
+		fmt.Sprintf("/repos/%s/%s/releases/%d/assets", user, repo, id),
+		nil, nil, &r)
+	return r, err
+}
+
+// GetReleaseAsset gets all the assets of a release in a repository
+func (c *Client) GetReleaseAsset(user, repo string, releaseId int64, assetId int64) (*Release, error) {
```

Comment:
releaseId must be called releaseID, assetId -> assetID

Run `make lint` to check if your golang code validates against GoLang naming convention